### ISSUE TITLE
Datastore - Initial Save with QueryPredicate.All should succeed 

### ISSUE THREAD

**Issue by fjnoyp** [aws-amplify/amplify-ios#1636]
### Describe the bug

When making an initial save with a queryPredicate .all the save fails saying a condition cannot be passed.  

This is different than Amplify Android that allows the initial save to succeed as queryPredicate .all is already equivalent to not having a save condition. 

### Steps To Reproduce

```swift
Call initial save with query predicate .all

Save fails with error
```

### Expected behavior

Initial save should succeed

### Amplify Framework Version

Latest

### Amplify Categories

DataStore

### Dependency manager

Cocoapods

### Swift version

?

### CLI version

.

### Xcode version

.

### Is this a regression?

No

### Device

.

### iOS Version

.

**Comment by lawmicha** [comment]
Android's logic https://github.com/aws-amplify/amplify-android/blob/main/aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java#L339-L351

**Comment by diegocstn** [comment]
I'm optimistically closing this as a fix has been released in [Amplify v1.20.0](https://github.com/aws-amplify/amplify-ios/releases/tag/v1.20.0). @fjnoyp let us know if that solved your issue!
